tests/integration_tests/test_mlflow.py

```diff
@@ -27,8 +27,8 @@ def _objective_func(trial: optuna.trial.Trial) -> float:
 
 def _multiobjective_func(trial: optuna.trial.Trial) -> Tuple[float, float]:
 
-    x = trial.suggest_float("x", low=-10, high=10)
-    y = trial.suggest_float("y", low=1, high=10, log=True)
+    x = trial.suggest_float("x", low=-1.0, high=1.0)
+    y = trial.suggest_float("y", low=20, high=30, log=True)
```

Comment:
Same as `_objective_func` since `enqueue_trial` is called with same parameters for both objectives in `test_log_metric`.